### ISSUE TITLE
undefined window on SSR 

### ISSUE THREAD

**Issue by Chalkin** [xon52/x5-gmaps#60]
When running as SSR you get this undefined window error and a 500 Internal Error on the server.

Project is created via vue-cli

```
error during render : /
ReferenceError: window is not defined
    at loaded ( /node_modules/x5-gmaps/dist/x5-gmaps.ssr.js:299:3)
    at init ( /node_modules/x5-gmaps/dist/x5-gmaps.ssr.js:322:7)
    at Function.installX5Gmaps [as install] ( /node_modules/x5-gmaps/dist/x5-gmaps.ssr.js:3101:36)
    at Function.Vue.use ( /node_modules/vue/dist/vue.runtime.common.dev.js:5087:22)
    at Module.a576 (src/main.js:100:14)
    at a (webpack/bootstrap:36:26)
    at module.exports.0463 (src/components/layout/navigation/MenuDesktop.vue?b699:1:0)
    at Object.<anonymous> (src/components/layout/navigation/MenuDesktop.vue?b699:1:0)
    at evaluateModule ( /node_modules/vue-server-renderer/build.dev.js:9322:21)
    at  /node_modules/vue-server-renderer/build.dev.js:9380:18
```

**Implementation:**

**Parent/Wrapper Component**

```
<template>
    <div class="c3-google-maps" :style="styles">
        <gmaps-map :options="mapOptions">
            <gmaps-marker :options="markerOptions" />
        </gmaps-map>
    </div>
</template>

<script>
import { gmapsMap, gmapsMarker } from 'x5-gmaps';

export default {
  name: 'GMaps',

  components: {
    gmapsMap,
    gmapsMarker,
  },

  props: {
    width: {
      type: String,
      required: false,
      default: '500px',
    },
    height: {
      type: String,
      required: false,
      default: '500px',
    },
    mapOptions: {
      type: Object,
      required: true,
    },
    markerOptions: {
      type: Object,
      required: true,
    },
  },

  computed: {
    styles() {
      return {
        width: this.width,
        height: this.height,
      };
    },
  },
};
</script>

<style scoped lang="scss">
    .c3-google-maps {
        width: 100%;
        height: 500px;
    }
</style>
```

**Parent:**

```
<div>
                <g-maps
                    width="700px"
                    height="500px"
                    :marker-options="{ position: { lat: 47.2983606, lng: 10.9810179 } }"
                    :map-options="{ center: { lat: 47.2988888, lng: 10.9800412 }, zoom: 15, mapTypeId: 'roadmap', }"
                />
            </div>
```

**Comment by xon52** [comment]
Hey thanks for reporting this. Apologies I missed it.
And in advance, I am also not very experienced in SSR so I'm hoping you can help try a few things for me to add to the docs around SSR use.

I have made a few minor changes in this PR #62 and published ~~v0.5.3~~ v0.5.4 for you to try.

I have seen people wrap components in `<no-ssr>` and `<client-only>` tags to avoid this, but a) that means it won't actually render on the SS, and b) I am not sure if that will solve the problem if it's coming from the _init script_ (which adds the actual Google Maps script to the `<head>`).

Keen for other thoughts on solving this though!

**Comment by Chalkin** [comment]
That version seems to resolve the issue. Thanks a lot :)
